app/src/widgets/totd-world-countries/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+
+import useTriggerUpdate from "common/hooks/useTriggerUpdate";
+
+import { WidgetProps } from "../index";
+
+export { saga } from "./sagas";
+
+const TotdWorldCountries: React.FC<Props> = ({
+  id,
+  name,
+  capital,
+  countryCode,
+  currency,
+  languages,
+  tld,
+  flag,
+  meta,
+  triggerUpdate,
+}) => {
+  useTriggerUpdate({ id, params: {}, meta, triggerUpdate }, []);
+
+  return (
+    <div>
+      <div className="text-3 w-100 p-2 text-center">
+        <u>
+          <strong>{name}</strong>
+        </u>
+      </div>
+      <div className="text-2">
+        <strong>Capital: </strong>
+        {capital}
+      </div>
+      <div className="text-2">
+        <strong>Country Code: </strong> {countryCode}
+      </div>
+      <div className="text-2">
+        <strong>Currency: </strong>
+        {currency}
+      </div>
+      <div className="text-2">
+        <strong>Languages: </strong>
+        {languages}
+      </div>
+      <div className="text-2">
+        <strong>Top-Level-Domain: </strong>
+        {tld}
+      </div>
+      {flag !== null && (
```

Comment:
```suggestion
      {flag && (
```

Today is the perfect day to test it, because "British Virgin Islands" do not have a dedicated flag:

![image](https://user-images.githubusercontent.com/3101914/96876182-6f3f9100-1478-11eb-947f-209ab0a7568d.png)